bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyComponents.java

```diff
@@ -292,7 +275,7 @@ public static boolean updateMeters(ShellyThingInterface thingHandler, ShellySett
                             updated |= thingHandler.updateChannel(groupName, CHANNEL_EMETER_VOLTAGE,
                                     toQuantityType(getDouble(emeter.voltage), DIGITS_VOLT, Units.VOLT));
                             updated |= thingHandler.updateChannel(groupName, CHANNEL_EMETER_CURRENT,
-                                    toQuantityType(getDouble(emeter.current), DIGITS_VOLT, Units.AMPERE));
+                                    toQuantityType(getDouble(emeter.current), DIGITS_KWH, Units.AMPERE));
```

Comment:
Please introduce a new `DIGITS_AMPERE` constant, or rename to `DIGITS_3` if you don't want the abstraction.

Reply:
new DIGITS_AMPERE